client/src/pages/chatlink/components/button.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const Button = (props) => {
+  
+  const { label, type, state, onClick, width } = props
+  
+  const styles = {
+    background: type == 'secondary' ? '#4caf50' : '#3944bc',
+    color: '#fff',
+    padding: '10px',
+    width: width,
```

Comment:
`width` should be optional. Often time we would want the button to take the width automatically based on the content.

Suggestion:
```
width: width || 'auto',
display: inline-block
```
`alignItem`, `justifyContent` - not really required in this case..